pkg/cloudprovider/vm.go

```diff
@@ -14,8 +15,15 @@ import (
 	"github.com/golang/glog"
 )
 
+const (
+	getVmOperationName = "vm_get"
```

Comment:
Can we please move this to single package location ?

Reply:
Shouldn't we keep operations name local to package wherever they are used ?
Shall we move all operations name to package metrics ?